changelog/update-7299-general-settings-desc

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Minor copy improvements on the Settings page.
```

Comment:
Suggest making this specific since it's a targeted improvement – e.g. `Remove mention of test mode from general settings help text`. Though probably can skip changelog for this.

Reply:
That makes sense. I'll update the changelog.

Reply:
Updated in https://github.com/Automattic/woocommerce-payments/pull/8410/commits/8a0707c91202c28f26d1a96ab300cb90cf96381c.